google/cloud/internal/grpc_request_metadata.cc

```diff
@@ -66,10 +68,15 @@ void AddContextMetadata(grpc::ClientContext const& context,
 
 }  // namespace
 
-RpcMetadata GetRequestMetadataFromContext(grpc::ClientContext const& context) {
+RpcMetadata GetRequestMetadataFromContext(grpc::ClientContext const& context,
+                                          internal::ErrorOrigin error_origin) {
   RpcMetadata metadata;
   AddContextMetadata(context, metadata);
-  AddServerRequestMetadata(context, metadata);
+  // If there was a client-originated error, the request never made it to the
+  // server metadata. Therefore, it doesn't exist so we should skip the call.
```

Comment:
Perhaps clarify "it".

Reply:
> "the request never made it to the server metadata"

I do not understand what this means. I think you are trying to say "the initial server metadata was never received by the client".

Reply:
Done